<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-B5LLFCD0" failures="3" tests="4" name="Test(failed)" time="45.261" errors="0" timestamp="2022-03-10T17:15:43 IST">
  <testcase classname="com.inetbank.testCase.TC_LoginTest_001" name="loginTest" time="3.303"/>
  <testcase classname="com.inetbank.testCase.TC_LoginTestDDT_002" name="loginDDT" time="6.526">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbank.testCase.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbank.testCase.TC_LoginTestDDT_002" name="loginDDT" time="8.09">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbank.testCase.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbank.testCase.TC_LoginTestDDT_002" name="loginDDT" time="7.155">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbank.testCase.TC_LoginTestDDT_002.loginDDT(TC_LoginTestDDT_002.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- Test(failed) -->
